salt/environment_settings.yml

```diff
@@ -141,19 +149,22 @@ environments:
           edx-worker:
             number: 1
             type: t2.medium
-      sandbox:
+      sandbox-residential-live:
```

Comment:
What's the reason for changing the purpose name?

Reply:
Figured it would be consistent with the overall naming of residential instances and thought that we have cases were we are filtering on `type` and needed to have that specified otherwise it would mean more code refactoring.

Reply:
I think that most of the cases around type are for things like the orchestration for deploy and ELB. For the sandbox I think we're going to just use the apps.deploy orchestrate command.

Reply:
Wouldn't we need to add an entry in pillar for sandbox if we revert the name back to sandbox from `sandbox-residential-live`?
https://github.com/mitodl/salt-ops/blob/master/pillar/top.sls#L159